x-pack/filebeat/input/httpjson/input_test.go

```diff
@@ -389,6 +392,195 @@ func TestInput(t *testing.T) {
 			handler:  defaultHandler("GET", ""),
 			expected: []string{`{"hello":[{"world":"moon"},{"space":[{"cake":"pumpkin"}]}]}`},
 		},
+		{
+			name:        "Test simple Chain GET request",
+			setupServer: newChainTestServer(httptest.NewServer),
+			baseConfig: map[string]interface{}{
+				"interval":       10,
+				"request.method": "GET",
+				"chain": []interface{}{
+					map[string]interface{}{
+						"step": map[string]interface{}{
+							"request.method": "GET",
+							"replace":        "$.records[:].id",
+						},
+					},
+				},
+			},
+			handler:  defaultHandler("GET", ""),
```

Comment:
It's not complaining.
should I update, now?

Reply:
It must be below the threshold (3 instances). For consistency with the rest of the package and to avoid it tipping when another one is added, I think it should be changed (all instances).

Reply:
Well, looking ... I'm wrong, there are 18 instance. I don't know why it's not complaining, but it should be changed anyway.